File: mambo-protocol/src/main/scala/org/mambo/protocol/enums/CharacterCreationResultEnum.scala




















// Generated on 05/02/2013 20:27:17
// Scala version by Blackrush
package org.mambo.protocol.enums

import org.mambo.protocol.Enumeration

object CharacterCreationResultEnum extends Enumeration {
  val ok = Value(0, "ok")
  val errNoReason = Value(1, "errNoReason")
  val errInvalidName = Value(2, "errInvalidName")
  val errNameAlreadyExists = Value(3, "errNameAlreadyExists")
  val errTooManyCharacters = Value(4, "errTooManyCharacters")
  val errNotAllowed = Value(5, "errNotAllowed")
  val errNewPlayerNotAllowed = Value(6, "errNewPlayerNotAllowed")
  val errRestricedZone = Value(7, "errRestricedZone")
}

File: mambo-protocol/src/main/java/org/mambo/protocol/enums/CharacterCreationResultEnum.java


// Generated on 05/08/2013 19:04:44
package org.mambo.protocol.enums;

public enum CharacterCreationResultEnum {
    OK(0),
    ERR_NO_REASON(1),
    ERR_INVALID_NAME(2),
    ERR_NAME_ALREADY_EXISTS(3),
    ERR_TOO_MANY_CHARACTERS(4),
    ERR_NOT_ALLOWED(5),
    ERR_NEW_PLAYER_NOT_ALLOWED(6),
    ERR_RESTRICED_ZONE(7);
    
    private int value;
    CharacterCreationResultEnum(int value) { this.value = value; }
    public int value() { return value; }
    
    public static CharacterCreationResultEnum valueOf(int value) {
    	for (CharacterCreationResultEnum a : values()) {
    		if (a.value == value)
    			return a;
    	}
    	return null;
    }
    
}
